wallets/metamask/src/utils/waitFor.ts

```diff
@@ -21,6 +22,60 @@ export const waitForSelector = async (selector: string, page: Page, timeout: num
   }
 }
 
+export const waitForMetaMaskLoad = async (page: Page) => {
+  await Promise.all(
+    LoadingSelectors.loadingIndicators.map(async (selector) => {
+      await waitForSelector(selector, page, 5000)
```

Comment:
it will be easier to specify default timeout (as for most of the cases it will be enough to use the function)

Reply:
Created DEFAULT_TIMEOUT and set it to 2000